### PR TITLE
Add test cases and examples for kubernetes

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description
Add test cases and examples for kubernetes.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    It 'DeleteViaIdentity' {
+        $connaks = Get-AzConnectedKubernetes -ResourceGroupName $env.resourceGroup -Name $env.connaksName01
+        Remove-AzConnectedKubernetes -InputObject $connaks
+        $connaksList =  Get-AzConnectedKubernetes -ResourceGroupName $env.resourceGroup
+        $connaksList.Name | Should -Not -Contain $env.connaksName01
+    }
+    BeforeAll {
+        helm delete azure-arc --no-hooks
+    }
```
Comment (wyunchi-ms) [inline src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1:30]: This section should be in `New-AzConnectedKubernetes`.

File: src/ConnectedKubernetes/test/New-AzConnectedKubernetes.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Describe 'New-AzConnectedKubernetes' {
+    It 'CreateExpanded' {
+        New-AzConnectedKubernetes -ClusterName $env.connaksName02 -ResourceGroupName $env.resourceGroup -Location $env.location -KubeConfig $HOME\.kube\config -KubeContext $env.kubeContext
+        $connaks = Get-AzConnectedKubernetes -ResourceGroupName $env.resourceGroup -Name $env.connaksName02
+        $connaks.ProvisioningState | Should -Be 'Succeeded'
+
+        New-AzConnectedKubernetes -ClusterName $env.connaksName03 -ResourceGroupName $env.resourceGroup -Location $env.location 
+        $connaks = Get-AzConnectedKubernetes -ResourceGroupName $env.resourceGroup -Name $env.connaksName03
+        $connaks.ProvisioningState | Should -Be 'Succeeded'
+    }
```
Comment (wyunchi-ms) [inline src/ConnectedKubernetes/test/New-AzConnectedKubernetes.Tests.ps1:23]: `New-AzConnectedKubernetes` and `Remove-AzConnectedKubernetes` should be in `live` mode and it's not nessary to checkin the record file.

File: src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,31 @@
+$loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
```
Comment (wyunchi-ms) [inline src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1:1]: `New-AzConnectedKubernetes` and `Remove-AzConnectedKubernetes` should be in `live` mode and it's not nessary to checkin the record file.

**Comment by wyunchi-ms** [comment]
You can skip the `live` mode test case for now. @LucasYao93 .

File: src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1 (hunk 1 above)
Comment (LucasYao93) [reply src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1]: Moved to New-AzConnectedKubernetes

File: src/ConnectedKubernetes/test/New-AzConnectedKubernetes.Tests.ps1 (hunk 2 above)
Comment (LucasYao93) [reply src/ConnectedKubernetes/test/New-AzConnectedKubernetes.Tests.ps1:26]: Add Skip tag for test cases for New-AzConnectedKubernetes and Remove-AzConnectedKubernetes

File: src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1 (hunk 3 above)
Comment (LucasYao93) [reply src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1:1]: Add Skip tag for test cases for New-AzConnectedKubernetes and Remove-AzConnectedKubernetes

**Pull request merged**